### PR TITLE
Feature/LP56 dashboard UI

### PR DESCRIPTION
**Pull request by Lupillo17**
Finished: DashboardView
* view finished 
* added mock data only for test purpose 
* added string.xml file

Modified: LpGenericCard
* deleting modifier padding in the card
* accountNumber text padding modified and just apply at the end of the text
* accountNumber align text at the end of content
* accountNumber max lines = 1

UI example:
image

### CODE REVIEW

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import com.iteneum.dashboard.R
+import com.iteneum.designsystem.components.LPGenericElevatedCard
+import com.iteneum.designsystem.components.LpBadgeButton
+import com.iteneum.designsystem.components.LpGenericCard
+import com.iteneum.designsystem.components.getRandomColor
+import com.iteneum.designsystem.theme.LPTypography
+import com.iteneum.designsystem.theme.LeasePertTheme
+import kotlin.random.Random
+
+@OptIn(ExperimentalMaterial3Api::class)
```
Comment (carlosph816) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:36]: You need to put the class information please

File: feature/example/build.gradle [hunk 2]
```diff
@@ -40,4 +40,5 @@ dependencies {
 
     //Mudules
     implementation project(":core:designsystem")
+    implementation project(":feature:dashboard")
```
Comment (irv205) [inline feature/example/build.gradle]: remove this implementation, only use this when you need to test your view, after that remove.

File: feature/dashboard/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
Comment (irv205) [inline feature/dashboard/src/main/res/values/strings.xml:2]: You can remove "dashboard", as each String file is individual it is not necessary to make more specifications

File: feature/example/build.gradle (hunk 2 above)
Comment (Lupillo17) [reply feature/example/build.gradle]: solving

Comment (Lupillo17) [reply feature/example/build.gradle]: Comment solved

File: feature/dashboard/src/main/res/values/strings.xml (hunk 3 above)
Comment (Lupillo17) [reply feature/dashboard/src/main/res/values/strings.xml:2]: Comment solved

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt (hunk 1 above)
Comment (Lupillo17) [reply feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:36]: Comment solved

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    val currentBalance = "0.00"
+    val currentBalanceCurrency = true
+    val serviceRequest = "1"
+    val amenityReservations = "1"
+    val eventList = listOf("Coffee + Donuts", "Pool party", "Discounts")
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(sizes.regularSize)
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add padding atributes

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                    .weight(1f),
+                text = stringResource(R.string.welcome_back, username),
+                style = LPTypography.titleMedium,
+            )
+            LpBadgeButton(
+                badgeNumber = badgeNumberNotification,
+                showBadge = showBadgeNotification,
+                imageVector = Icons.Filled.Notifications
+            ) {
+                Log.e("tag", "go to notification")
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add a `//TODO` comment to tell us which action is going to be executed in this button.

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+            }
+            LpBadgeButton(
+                badgeNumber = badgeNumberPerson,
+                showBadge = showBadgePerson,
+                imageVector = Icons.Filled.Person
+            ) {
+                Log.e("tag", "go to person")
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add a `//TODO` comment to tell us which action is going to be executed in this button.

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+        Spacer(modifier = Modifier.size(sizes.minorRegularSize))
+        LpGenericCard(
+            modifier = Modifier
+                .fillMaxWidth(),
+            title = stringResource(R.string.current_balance),
+            details = stringResource(R.string.go_to_payments),
+            accountNumber = currentBalance,
+            currency = currentBalanceCurrency
+        ) {
+            Log.e("tag", "Go to payments")
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add a //TODO comment to tell us which action is going to be executed in this button.

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
+        LpGenericCard(
+            title = stringResource(R.string.service_request),
+            details = stringResource(R.string.view_detail),
+            accountNumber = serviceRequest,
+        ) {
+            Log.e("tag", "go to service request")
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add a //TODO comment to tell us which action is going to be executed in this button.

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+            }
+            LpBadgeButton(
+                badgeNumber = badgeNumberPerson,
+                showBadge = showBadgePerson,
+                imageVector = Icons.Filled.Person
+            ) {
+                Log.e("tag", "go to person")
+            }
+        }
+        Spacer(modifier = Modifier.size(sizes.minorRegularSize))
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Instead use a spacer I recommend you use the padding between the cards

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+            modifier = Modifier
+                .fillMaxWidth(),
+            title = stringResource(R.string.current_balance),
+            details = stringResource(R.string.go_to_payments),
+            accountNumber = currentBalance,
+            currency = currentBalanceCurrency
+        ) {
+            Log.e("tag", "Go to payments")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Instead use a spacer I recommend you use the padding between the cards

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+        LpGenericCard(
+            title = stringResource(R.string.service_request),
+            details = stringResource(R.string.view_detail),
+            accountNumber = serviceRequest,
+        ) {
+            Log.e("tag", "go to service request")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Instead use a spacer I recommend you use the padding between the cards

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+        LpGenericCard(
+            title = stringResource(R.string.amenity_reservations),
+            details = stringResource(R.string.view_detail),
+            accountNumber = amenityReservations
+        ) {
+            Log.e("tag", "go to amenity reservations")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Instead use a spacer I recommend you use the padding between the cards

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+        ) {
+            Log.e("tag", "go to service request")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
+        LpGenericCard(
+            title = stringResource(R.string.amenity_reservations),
+            details = stringResource(R.string.view_detail),
+            accountNumber = amenityReservations
+        ) {
+            Log.e("tag", "go to amenity reservations")
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add a //TODO comment to tell us which action is going to be executed in this button.

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            items(eventList) { event ->
+                LPGenericElevatedCard(
+                    title = event,
+                    description = "This is mock data",
+                    buttonText = "Go to Screen",
+                    color = getRandomColor(Random.nextInt(0, 3))
+                ) {
+                    Log.e("tag", event)
```
Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please add a //TODO comment to tell us which action is going to be executed in this button.

Comment (melissaquezada94) [inline feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Also I recommend you use a onClick high order function mock to sent in the event list

**Review by carlosph816 — APPROVED** [review]

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt (hunk 4 above)
Comment (Lupillo17) [reply feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: @melissaquezada94 all the comments solved

File: feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt (hunk 5 above)
Comment (melissaquezada94) [reply feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: Please remove the all the Logs and the import log

Comment (Lupillo17) [reply feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt]: solved

**Review by melissaquezada94 — APPROVED** [review]

**Pull request merged**